test/wasm2c/spec/i32.txt

```diff
@@ -1,5 +1,6 @@
 ;;; TOOL: run-spec-wasm2c
 ;;; STDIN_FILE: third_party/testsuite/i32.wast
+;;; SKIP: sign-extend, multi-value not supported yet
```

Comment:
Does wabt not support these?

Reply:
I don't get how these are marked as SKIP but still seem to report passing ?

Reply:
wasm2c doesn't support this yet. And `SKIP` doesn't run the test, but it also doesn't check the output. I guess that's a bit confusing, but it means the diffs here are smaller.

Reply:
Sorry I missed the sub directory name there.